dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/PageRenderedContext.java

```diff
@@ -0,0 +1,53 @@
+package com.dotmarketing.portlets.htmlpageasset.business.render;
+
+import com.dotmarketing.portlets.htmlpageasset.model.HTMLPageAsset;
+import com.dotmarketing.util.PageMode;
+import com.liferay.portal.model.User;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class PageRenderedContext {
+
+    private final HttpServletRequest request;
+    private final HttpServletResponse response;
+    private final User user;
+    private final String pageUri;
+    private final PageMode pageMode;
+    private final HTMLPageAsset page;
+
+    public PageRenderedContext(final HttpServletRequest request,
+            final HttpServletResponse response, final User user, final String pageUri,
+            final PageMode pageMode, final HTMLPageAsset page) {
+        this.request = request;
+        this.response = response;
+        this.user = user;
+        this.pageUri = pageUri;
+        this.pageMode = pageMode;
+        this.page = page;
```

Comment:
Add the langaugeId

Reply:
@wezell for this change we need to do a bigger refactoring to pass the language_Id in several methods of this API HTMLPageAssetRenderedAPI and not take this from the request, how we do here for example:

https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/HTMLPageAssetRenderedAPIImpl.java#L188

Also we have to remove the HttpRequest as parameter from several method of this class too